docs/contribution.md

```diff
@@ -34,6 +34,10 @@ Before proposing changes, look for similar ones in the project's [issues](https:
   * Bash is essential for running zx scripts. Linux and macOS users usually have it installed by default. Consider using [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install) or [Git Bash](https://git-scm.com/downloads) if you are on Windows.
 * Fork [the repository](https://github.com/google/zx).
 * Create a new branch.
+* Switch to the recommended version of Node.js
+  * If you are using [Node.js version manager](https://github.com/shadowspawn/node-version-usage) supported `.node_version` using it.
+  * If you are using Volta, Node.js version will be set automatically from `package.json`.
+  * In another case manually install recommended version from  `.node_version` file.
```

Comment:
* Install manually Node.js >= 22
* Delegate the routine to any version manager, that [supports `.node_version` config](https://stackoverflow.com/questions/27425852/what-uses-respects-the-node-version-file)
* Use Volta, the target version will be set automatically from the `package.json`

Reply:
D1